package-lock.json

```diff
@@ -8251,7 +8251,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
+          "resolved": "https://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```

Comment:
Yes, I know these changes are unrelated. However, every single time I touch the webapp repo, my node/npm version makes these changes. Would be good to get this in with a small PR. 0/5.

Reply:
Interesting mine converts to `http` again 🤷

Reply:
Lol, probably you need to upgrade Node.